core/frontend-stubs/cli-frontend-stub/src/main/java/org/eclipse/sw360/antenna/frontend/stub/cli/AntennaCLISettingsReader.java

```diff
@@ -203,4 +209,22 @@ public ToolConfiguration readSettingsToToolConfiguration(XmlSettingsReader reade
 
         return toolConfigBuilder.buildConfiguration();
     }
+
+
+    private Map<String, Object> mapProperties(File propertiesFile) {
+        try (InputStream input = new FileInputStream(propertiesFile)) {
+            Properties prop = new Properties();
+
+            prop.load(input);
+
+            return prop.entrySet()
+                    .stream()
+                    .collect(Collectors.toMap(
+                            p -> p.getKey().toString(),
+                            p -> p.getValue().toString()
+                    ));
+        } catch (IOException e) {
+            throw new IllegalArgumentException("IO exception when reading properties file: " + e.getMessage());
```

Comment:
`IllegalArgumentException` is not correct here:

> Thrown to indicate that a method has been passed an illegal or inappropriate argument.

but the argument `propertiesFile` was valid and the function was not used incorrectly.

Reply:
I don't know why this is shown as changed, but this is already like this:
https://github.com/eclipse/antenna/blob/fc457dd0829e0dfe5aad2bd151bafe1abadba8e1/core/frontend-stubs/cli-frontend-stub/src/main/java/org/eclipse/sw360/antenna/frontend/stub/cli/AntennaCLISettingsReader.java#L138

What do you propose otherwise?

Reply:
Then this is also duplicated code from line 142ff